src/shared/utilities/pathUtils.ts

```diff
@@ -34,3 +67,14 @@ export function getLocalRootVariants(filePath: string): string[] {
 
     return [filePath]
 }
+
+/**
+ * "c:/foo.bar" => "/foo.bar"
+ */
+export function removeDriveLetter(f: string): string {
```

Comment:
What is this for? It's only used in tests; I can't think of any practical reason to strip the Windows drive letter.

Reply:
some paths in the config don't have a drive letter. I didn't dig into why, yet. This can be revised later.

created a tracking issue.